stable/insights-agent/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart to run the Fairwinds Insights agent
 name: insights-agent
-version: 2.0.7
+version: 2.0.8
```

Comment:
Should this be a larger version bump so customers do not surprise-get OPA 2.0 after having run agent 2.0 for a while?